app/assets/stylesheets/themes/dark.css

```diff
@@ -1,3 +1,8 @@
+::selection {
+  background-color: rgba(248,28,229, 0.5); /* 0.5 represents 50% opacity */
+  color: #fff;
```

Comment:
@rt4914 if you change `color: #fff;` to `rgba(255,255,255,0.96);` it will work on android. It's something related to [this]( https://stackoverflow.com/a/7224621/7404975): 

> For some reason Chrome forces it to be semi-transparent. However, you can get around this by setting the background using rgba